azure/services/resourceskus/sku.go

```diff
@@ -128,3 +130,23 @@ func (s SKU) GetCapability(name string) (string, bool) {
 	}
 	return "", false
 }
+
+// HasLocationCapability returns true if the provided resource supports the location capability.
+func (s SKU) HasLocationCapability(name, size, location string) bool {
+	if s.LocationInfo != nil && *s.Name == size {
+		for _, info := range *s.LocationInfo {
+			if info.ZoneDetails != nil && *info.Location == location {
+				for _, zone := range *info.ZoneDetails {
+					if zone.Capabilities != nil {
+						for _, capability := range *zone.Capabilities {
+							if *capability.Name == name {
```

Comment:
very technically (I've never seen it to be the case), `*capability.Value` can be false. see other funcs using it.

Reply:
sure we can add the check for Value as true

Reply:
@alexeldeib please have look at the suggestion commit